src/IntaRNA/Accessibility.cpp

```diff
@@ -153,7 +158,7 @@ compute_accessible_ranges_recursive( IndexRangeList& ranges, const size_t max_se
 					compute_accessible_ranges_recursive(ranges, max_seq_length, ED, from, to);
 				}
 				else{
-					ranges.push_back(IndexRange(from+1,to+1));
+					ranges.push_back(IndexRange(from,to));
```

Comment:
huuh.. the index-start-1-convention only applies to user input! internally you should use index-start-0 of course.. uaks.. sorry for the confusion!